packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ExpressiveModal } from '../ExpressiveModal';
+import { ModalBody } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { DDS_LIGHTBOX_MEDIA_VIEWER } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * LightboxMediaViewer Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title LightboxMediaViewer media title
+ * @param {string} props.copy LightboxMediaViewer media short description
+ * @param {string} props.image LightboxMediaViewer responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
+  /**
+   * Set default image
+   *
+   * @param {*} image responsive image object
+   * @returns {string} default image url
+   */
+  const defaultImage = image => {
+    return image.uri.md
+      ? image.uri.md
+      : image.uri.lg
+      ? image.uri.lg
+      : image.uri.sm;
+  };
+
+  if (!image.uri.lg) {
+    return null;
+  }
+  return featureFlag(
+    DDS_LIGHTBOX_MEDIA_VIEWER,
+    <section
+      data-autoid={`${stablePrefix}--lightbox-media-viewer`}
+      className={`${prefix}--lightbox-media-viewer`}>
+      <ExpressiveModal open={open} fullwidth={true} onClose={onClose}>
```

Comment:
I wonder if it would reduce complexity in documentation, and possibly code if we just use an object spread and allow users to pass in props based on the `ExpressiveModal `dependency documentation.

```jsx
<ExpressiveModal fullwidth={true} {...modalProps}>
```